genie-core/src/main/java/com/netflix/genie/core/properties/JobsUsersActiveLimitProperties.java

```diff
@@ -46,4 +48,9 @@
     private boolean enabled = DEFAULT_ENABLED;
     @Min(value = 1)
     private int count = DEFAULT_COUNT;
+    private Map<String, Integer> specialUser = Maps.newHashMap();
+
+    public int getUserLimit(final String user) {
+        return specialUser.getOrDefault(user, count);
```

Comment:
I see in this diff that `count = DEFAULT_COUNT`. I assume somewhere else in here it is binding the value provided in the properties file that could overwrite that?

Reply:
It is initialized to default, but when spring properties evaluate and bind values it can set it to something else (via generated setter).